pkg/machines/scripts/create_machine.sh

```diff
@@ -72,10 +72,11 @@ fi
 XMLS_FILE="`mktemp`"
 
 if [ "$START_VM" = "true" ]; then
-    # Use --wait 0 to not wait till guest console closes.
-    # Simply kick off the install and exit
-    STARTUP_PARAMS="--noautoconsole --wait 0"
+    STARTUP_PARAMS="--noautoconsole"
     HAS_INSTALL_PHASE="false"
+    if [ "$SOURCE_TYPE" != "disk_image" ]; then
```

Comment:
I wonder if that should be stricter, only for isos and repo URLs. E. g. when you PXE boot, you also don't want to wait for an installer, as very often you'd boot the actual OS that way, not an installer?

Reply:
It makes sense. So I will adjust it to wait for the installer to finish only in case the source is ISO or distro tree URL.